doc/nrf/ug_thread_power_consumption.rst

```diff
@@ -0,0 +1,100 @@
+.. _thread_power_consumption:
+
+OpenThread power consumption
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page provides information about the amount of power used by Sleepy End Devices (SEDs) and Synchronized Sleepy End Devices (SSEDs) that use the OpenThread stack.
+
+.. _thread_power_consumption_methodology:
+
+Measurements methodology
+************************
+
+The measurement setup consists of:
+
+* An nrf52840dk_nrf52840 board used as a Thread leader.
+* A DUT board used as a Thread child.
+* A `Power Profiler Kit II (PPK2)`_ attached to the DUT according to the instructions in its Quick start guide.
+
+The leader board is flashed with the regular :ref:`Thread CLI sample <ot_cli_sample>` firmware.
+The DUT board is flashed with the :ref:`Thread CLI sample low power mode <ot_cli_sample_low_power>` firmware.
```

Comment:
Would it make sense to enable RAM Power Down lib: https://github.com/nrfconnect/sdk-nrf/blob/main/lib/ram_pwrdn/Kconfig#L7

Reply:
This is already done: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/openthread/cli/overlay-low_power.conf#L9, https://github.com/nrfconnect/sdk-nrf/blob/main/samples/openthread/cli/src/low_power.c#L27

I'll add the building instructions to make clear which code is being used.